libdirections-offboard/src/test/java/com/mapbox/navigation/route/offboard/MapboxOffboardRouterTest.kt

```diff
@@ -19,26 +19,22 @@ import retrofit2.Response
 
 class MapboxOffboardRouterTest : BaseTest() {
 
-    companion object {
-        private const val ACCESS_TOKEN = "pk.token"
-    }
-
     private val navigationRoute = mockk<NavigationRoute>(relaxed = true)
+    // private val navigationRoute = NavigationRoute.builder(mockk(relaxed = true)).build()
```

Comment:
NIT Should we remove this commented out code?

Reply:
removed

Reply:
Was marked as resolved but the comment is still here. Did you push :trollface:?